docs/ksp-additional-details.md

```diff
@@ -0,0 +1,62 @@
+# How KSP models Kotlin code
+
+The API definition can be found [here](api/src/main/kotlin/com/google/devtools/ksp/).
```

Comment:
../api/src/main/kotlin/com/google/devtools/ksp/

Reply:
Thanks for the catch, updated!